src/components/ui/TripCard/AccordianDetails/TripCardDetails.tsx

```diff
@@ -0,0 +1,83 @@
+import { AccordionDetails, IconButton, Stack, Tooltip } from '@mui/material';
+import { DetailsGrid } from './DetailsGrid/DetailsGrid';
+import SeatLayout from '../../../SeatLayout/SeatLayout';
+import { useState } from 'react';
+import { layoutNames, seats } from '../../../SeatLayout/seatConfig';
+import { SeatLegend } from './SeatLegend/SeatLegend';
+import { StyledAlert } from '../../../Alert/Alert.styled';
+import { Checkout } from './Checkout/Checkout';
+import { useTranslation } from 'react-i18next';
+import DeleteForeverIcon from '@mui/icons-material/DeleteForever';
+
+export const TripCardDetails = () => {
+    const [selectedSeats, setSelectedSeats] = useState<number[]>([]);
+
+    const updateSelectedSeats = (newSeat: number) => {
+        if (selectedSeats.includes(newSeat)) {
+            setSelectedSeats((prev) =>
+                prev.filter((selectedSeat) => selectedSeat != newSeat)
+            );
+        } else setSelectedSeats((prev) => [...prev, newSeat]);
+    };
+    const clearSelectedSeats = () => {
+        setSelectedSeats([]);
+    };
+
+    const { t } = useTranslation('tripDetails');
+
+    return (
+        <AccordionDetails>
+            <Stack direction={'column'} p={3} pt={0} spacing={4}>
```

Comment:
Provide more white space within the accirdion by adding paddin between accordion title and content and bus layout.

Reply:
Resolved in 5137345a6671304eb7b016e929f5bec212e570c6